bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -29,12 +29,13 @@ spec:
     or by specifying existing instances through a [ConfigMap](https://docs.openshift.com/container-platform/latest/windows_containers/configuring-byoh-windows-instance.html)
     The operator completes all the necessary steps to configure the Windows instance so that it can join the cluster as a worker node.
 
-    Usage of this operator requires a Red Hat OpenShift subscription. Users looking to deploy Windows containers workloads in
-    production clusters should acquire a subscription before attempting to install this operator. Users without a subscription
-    can try the community operator, a distribution which lacks official support.
+    Usage of this operator requires an OpenShift Container Platform for Windows Containers subscription. Users looking to
+    deploy Windows containers workloads in production clusters should acquire a subscription before attempting to
+    install this operator. Users without a subscription can try the community operator, a distribution which lacks
+    official support.
 
     ### Pre-requisites
-    * A Red Hat OpenShift subscription
+    * A Red Hat OpenShift Container Platform for Windows Containers subscription
```

Comment:
Would it help to add a link to the support page here: https://access.redhat.com/support/policy/updates/openshift#windows